src/VirtoCommerce.Platform.Web/wwwroot/js/app/userProfile/blades/userProfile.js

```diff
@@ -53,6 +53,14 @@ angular.module('platformWebApp')
                 blade.currentTimeZone = getNameByCode($scope.timeZones, blade.currentTimeZone);
                 blade.currentTimeAgoSettings = userProfile.timeAgoSettings;
                 blade.currentTimeSettings = userProfile.timeSettings;
+
+                if (userProfile.memberId && _.any(moduleHelper.modules, module => module.id === 'VirtoCommerce.Customer')) {
+                    $http.get('api/members/' + userProfile.memberId).then(function (memberResponse) {
+                        blade.currentEntity = memberResponse.data;
+                        // This flag is used to save icon immediately in the MemberIcon blade of the Customer model
+                        blade.saveIconImmediately = true;
+                    });
+                }
```

Comment:
Patform shoudn't know about specific module and shouldn't call its API directly.